.github/workflows/build-comms-api.yml

```diff
@@ -0,0 +1,64 @@
+name: Build Comms API
+
+on:
+  push:
+    branches: [ "otter" ]
```

Comment:
this should be just

on:
  push:
      paths:
      - packages/server/customer-os-common-module/**
      - packages/server/comms-api/**
      - packages/server/customer-os-api/graph/schemas/**
      - .github/workflows/*-comms-api.yml